ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import {
+  ConfirmInfoRow,
+  ConfirmInfoRowText,
+  ConfirmInfoRowUrl,
+} from '../../../../../../components/app/confirm/info/row';
+import { useI18nContext } from '../../../../../../hooks/useI18nContext';
+import { currentConfirmationSelector } from '../../../../../../selectors';
+import { Box } from '../../../../../../components/component-library';
+import {
+  BackgroundColor,
+  BorderRadius,
+} from '../../../../../../helpers/constants/design-system';
+import { hexToText } from '../../../../../../helpers/utils/util';
+
+const PersonalSignInfo: React.FC = () => {
+  const t = useI18nContext();
+  const currentConfirmation = useSelector(currentConfirmationSelector);
+
+  if (!currentConfirmation?.msgParams) {
```

Comment:
is it possible for currentConfirmation to be undefined?

Reply:
TS complains if I do not use `?`